### ISSUE TITLE
Uncaught (in promise) TypeError: p[(p.length - 1)].lhs.hasOwnProperty is not a function

### ISSUE THREAD

**Issue by jhicken** [LogRocket/redux-logger#243]
So if i enable diff and my app throws an error I get this very unhelpful message. I haven't taken the time to dig in and find the cause. but It makes me lugubrious (translation) :(

**Comment by tonovotny** [comment]
Same here, downgrade to `3.0.1` helped.

**Comment by tonovotny** [comment]
Maybe it's caused by https://github.com/flitbit/diff/issues/102 (?)

**Comment by tonovotny** [comment]
Maybe it's caused by newer version of https://github.com/flitbit/diff (?)

**Comment by spencerbeggs** [comment]
I am having the same issue. Downgrading did not help. In desktop Safari this causing a memory overflow! I am investigating.

**Comment by djizco** [comment]
I'm having the same issue when I'm using routes with URL params. I think it could be something with the history stack.  It only occurs on routing actions. Verified downgrading from 3.x to 2.x fixed the issue.

**Comment by djizco** [comment]
i think @tonovotny is correct.  This pull requested seems to fix this issue in the diff package https://github.com/flitbit/diff/pull/114.  Until its fixed, redux-logger should just downgrade the dependency.

**Comment by sibelius** [comment]
any progress here?

**Comment by djizco** [comment]
I put in a pull request to use the correct version of diff, but code coverage reports are failing and it seems it hasn't been maintained in a while.

**Comment by sibelius** [comment]
@djizco which PR?

**Comment by renatobenks** [comment]
@evgenyrodionov please, this issue still need to fix!!!! ❌

**Comment by farskid** [comment]
I'm using redux-logger@3.0.6 and whenever my reducer or action creator throws an error, I get this error from the logger.
It also causes memory overflow sometimes.

**Comment by eranimo** [comment]
Yes, this issue still happens. We will likely be replacing this dependency.

**Comment by farskid** [comment]
@eranimo Anything I can help with to speed this up?

**Comment by eranimo** [comment]
@farskid If you submit a PR fixing it I would likely merge it 😄

**Comment by farskid** [comment]
@eranimo Any chance this has already been fixed? I tried to reproduce on the web and React native. no chance.
